indykite/resource_trust_score_profile.go

```diff
@@ -81,15 +81,15 @@ func resourceTrustScoreProfile() *schema.Resource {
 						trustScoreProfileName: {
 							Type:     schema.TypeString,
 							Required: true,
-							Description: "Name of the trust score dimensions. Possible values are: " +
-								strings.Join(getMapStringKeys(TrustScoreDimensionNames), ", ") + ".  " +
-								"Origin: Identifies where the data comes from, " +
+							Description: "Name of the trust score dimensions. Possible values are: `" +
```

Comment:
If you are changing this, run `tfdocs_generate`

Reply:
command run